java/kafka/pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <log4j.version>2.13.3</log4j.version>
+        <log4j.version>2.7</log4j.version>
```

Comment:
Why are we downgrading to older version? I would not expect that is something we want to do.

Reply:
Hmmm, that was version, which was suggested by Marko but i can try it with the `2.13.3`

Reply:
Fixed